### DISCUSSION TITLE
标点符号字体显示有误

### DISCUSSION THREAD

**Discussion by dreistein543** [nju-lug/NJUThesis#163] [Q&A]
使用版本: 1.0.0 (texlive 南大镜像站 可获得版本)
使用系统：macOS 12.6

字体设置：默认，
问题描述：
正文的标点符号，似乎被替换为楷体，但实际上应该是宋体标点。编译结果如下图
<img width="166" alt="截屏2022-10-24 19 52 31" src="https://user-images.githubusercontent.com/30745407/197519687-2cc5eeff-2cba-4d5e-9d1a-b2d2e8bd3072.png">

下图是我在word中分别展示楷体和宋体，可以发现，目前编译的结果中，正文的标点符号使用了楷体的字体，但本应该使用宋体字体。
<img width="281" alt="截屏2022-10-24 19 53 52" src="https://user-images.githubusercontent.com/30745407/197519985-6967ce2f-7de0-4614-98e0-0ff7763371fe.png">

尝试方案：
对于字体设置中 `cjk-font` 制定 mac 和 win，虽然不影响编译，但是结果并无改善。

**Comment by atxy-blip** [comment]
这个应该算 bug，转移至 issue 区讨论吧。我近期会在课题组借一台 mac 进行测试。
